src/runtime_src/core/include/xrt/xrt_pskernel.h

```diff
@@ -0,0 +1,713 @@
+/*
```

Comment:
The body and the header of xrt_pskernel.h have a lot of in-lining...Why is this needed?